wazo_call_logd/generator.py

```diff
@@ -54,3 +57,16 @@ def _get_interpretor(self, cels):
                 return interpretor
 
         raise RuntimeError('Could not find suitable interpretor in {}'.format(self._cel_interpretors))
+
+    @staticmethod
+    def _tenant_checker(call_log):
+        for prefix in ('requested', 'requested_internal',
+                       'source_internal', 'destination_internal'):
+            if getattr(call_log, '%s_tenant_uuid' % prefix):
```

Comment:
use `format`
I must admit that I don't really like to build the attribute name dynamically, it can be painful when doing `grep`. Have you an opinion? or my argument is invalid? :stuck_out_tongue: